src/FFmpegWriter.cpp

```diff
@@ -1803,9 +1851,13 @@ void FFmpegWriter::write_audio_packets(bool is_final, std::shared_ptr<openshot::
 
 			// Create output frame (and allocate arrays)
 			frame_final->nb_samples = audio_input_frame_size;
+#if HAVE_CH_LAYOUT
+			av_channel_layout_from_mask(&frame_final->ch_layout, info.channel_layout);
```

Comment:
Same question as before... I'm assuming we don't need to set channels manually anymore?

Reply:
Indeed. Here we get the mask from info.channel_layout, and create all ch_layout values from it.